src/tests/functional/plugin/conformance/test_runner/README.md

```diff
@@ -1,35 +1,56 @@
 # Conformance test runner
 
 ## Description
-Conformance suit is a set of tests with parameters independent from plug-in specific and limitations. It contains:
-* `ReadIR`. Allow to read IRs from folders recursive, infer it and compare results with reference.
-* `OpImplCheckTest`. Allow to check operation plugin implementation status (`Implemented`/`Not implemented`).
+Conformance suites certifies plugin functionality using a set of tests with plugin specificity independent parameters. There are 2 types of conformance validation.
 
-  **NOTE**: This test suite is in active development. The implementation status based on all other test results.
-* 
+### `API Conformance`
+The suite checks OpenVINO API the folowing entities a plugin implementation (Also there are test instantiations to validate hardware plugin functionality via software plugins (e.g. `MULTI`, `HETERO` and etc.) for the entities):
```

Comment:
The suite checks the following OpenVINO API entities in a plugin implementation:
		* plugin
		* compiled model (executable network)
		* infer request
Also, there are test instantiations to validate hardware plugin functionality via software plugins (e.g. `MULTI`, `HETERO` and etc.) for the entities.